packages/xod-cli/README.md

```diff
@@ -85,6 +87,68 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.1.0/src/commands/autocomplete/index.ts)_
 
+## `xodc boards [options]`
+
+show available boards
+
+```
+USAGE
+  $ xodc boards [options]
+
+OPTIONS
+  -V, --version         show CLI version
+  -h, --help            show CLI help
+  -q, --quiet           do not log messages other than errors
+  -w, --workspace=path  [default: ~/xod] use the workspace specified, defaults to $HOME/xod
+  --available           show available boards
```

Comment:
`xodc boards`, as states above, “shows available boards.” What does this flag adds. Looks like with it we “show available available boards”.

Do you mean “show only boards connected to PC”?

Reply:
Oops, overengineered. I removed these options. Just showing the table (as in the issue) with or without header (-q flag).